java/org/contikios/cooja/Cooja.java

```diff
@@ -1586,7 +1586,11 @@ Simulation createSimulation(Element root, boolean quick, boolean rewriteCsc, Lon
           mote = newSim.getMote(moteNr);
         }
       }
-      tryStartPlugin(pluginClass, newSim, mote, pluginElement);
+      try {
+        startPlugin(pluginClass, newSim, mote, pluginElement);
+      } catch (PluginConstructionException ex) {
+        throw new SimulationCreationException("Failed to start plugin: " + ex.getMessage(), ex);
```

Comment:
What do you think about including plugin name or plugin class name to inform which plugin failed to start?

Reply:
That should be included from #769.